src/main/java/it/pagopa/interop/probing/telemetry/writer/service/impl/TimestreamServiceImpl.java

```diff
@@ -33,7 +31,7 @@ public class TimestreamServiceImpl implements TimestreamService {
   @Autowired
   Logger logger;
 
-  private static final String NAME = "name";
+  private static final String ID = "id";
 
   private static final String KO_REASON = "ko_reason";
```

Comment:
Please be specific on constants naming, it's difficult to understand what `KO_REASON` is without looking how it is used in the code
```suggestion
  private static final String ESERVICE_RECORD_ID_DIMENSION = "eservice_record_id";

  private static final String KO_REASON_DIMENSION = "ko_reason";
```